### ISSUE TITLE
Bring back html report generation

### ISSUE THREAD

**Issue by Viss** [sensepost/gowitness#93]
In previous version of gowitness, outputting an html report was a feature, but it has since been removed. 

It's really handy when the resulting report is something that needs to be shared, and it would be really helpful to have the reporting component have the option to create files which users could just zip up and send around if the option to have multiple folks hit port 7171 on some host isnt availble - like in the instance where there is a vm in a remote environment where gowitness was run, and to have 'an entire team review the results', there is the overhead of cobbling together ssh tunnels or vpns to get people access to this remote asset. 

it's way simpler to just zip up a dir with an html report and screenshots and host it somewhere more accessible, or just drop it into some kind of collaboration tool

tl;dr - please bring back html report output <3

**Comment by leonjza** [comment]
Yeah we could add the `generate` command from v1 back again. In the meantime I suggest you just zip up the db & screenshots and have folks just `serve` locally to view the report, negating the need for everyone to connect to the server where gowitness originally ran.

**Comment by unl1k3ly** [comment]
+1 for report generate in a flat html file pointing to screenshots in a ./ path way.

**Comment by unl1k3ly** [comment]
similar way to aquatone for instance... output to a folder and have a report.html in there... :)

**Comment by sea-erkin** [comment]
I too miss the old report generation

**Comment by randomactsofsecurity** [comment]
Also, maybe add a button within the "serve" functionality to output that report as the HTML, besides the command line argument. And the html output should support Perception sorting too

**Comment by leonjza** [comment]
I added this again matching the style as well in https://github.com/sensepost/gowitness/commit/3e459eb0a10d10c80f929c8a387bb9fbc8ff6683. Will be available in the next tagged version.
